tests/e2e/project/browser.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { test, expect } from "@tests/support/fixtures.js";
+
+const sourceBrowsingFixture =
+  "/seeds/0.0.0.0/rad:git:hnrkgd7sjt79k4j59ddh11ooxg18rk7soej8o";
+
+test.describe("browser error handling", () => {
+  test("check that error is shown when not able to load folder", async ({
```

Comment:
It would be nice if we'd write the test subject so that it reads like a sentence.

Now:
`test("check that error is shown when not able to load folder"`

Test check that error is shown...

Suggestion:
`test("show error when folder can't be loaded"`

Test show error when folder can't be loaded.

Reply:
1ea4b49d